### ISSUE TITLE
Support for setting title attribute on embed element

### ISSUE THREAD

**Issue by androa** [pipwerks/PDFObject#248]
To work well with screen readers, it should be possible to set the [title attribute](https://developer.mozilla.org/en-US/docs/Web/HTML/Global_attributes/title) on the embed element that is created.

**Comment by pipwerks** [comment]
Yes, you're right. Thanks for the reminder

**Comment by pipwerks** [comment]
PDFObject 2.2.6 is live and includes a patch for this issue. Thanks for reporting, please let me know if you run into any problems.
